resources/dicts/patrols/plains/hunting/any.json

```diff
@@ -440,23 +512,151 @@
+                "exp": 20,
+                "weight": 20,
+                "art": "gen_hunt_burrow",
+                "can_have_stat": ["any"],
+                "stat_skill": ["CAMP,1"],
+                "prey": ["medium"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["trust"],
+                        "amount": 3
+                    },
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["respect"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "s_c makes sure that as the sun creeps over the horizon it flashes its light into the eyes of the patrol's prey. The trick wins both prey and purrs from {PRONOUN/s_c/poss} Clanmates.",
+                "exp": 20,
+                "weight": 20,
+                "art": "gen_hunt_rabbit",
+                "can_have_stat": ["any"],
+                "stat_skill": ["CLEVER,1", "HUNTER,1"],
+                "prey": ["medium"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["trust"],
+                        "amount": 3
+                    },
+                    {
+                        "cats_to": ["s_c"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["respect"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "Some days you win some - some days you lose some. It's nobody's fault.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_hunt_rabbit"
+            },
+            {
+                "text": "This warren was a bad call, and its rabbits twitch out of sight with the vigilence of the recently-hunted. Though p_l picks up a mouse on the way home, {PRONOUN/p_l/subject}{VERB/p_l/'re/'s} still disappointed.",
+                "exp": 0,
+                "weight": 20,
+                "art": "gen_hunt_sneakprey",
+                "prey": ["very_small"]
+            },
+            {
+                "text": "Bolds, s_c charges towards a feeding bunny. And sure, s_c gets <i>{PRONOUN/s_c/poss}</i> rabbit, but what about the rest of the patrol? It does not leave the rest of them feeling particularly impressed.",
```

Comment:
Bolds should be Boldly

Reply:
thank you so much for typo hunting! They should all be fixed now <3